src/pages/workspaces/List.js

```diff
@@ -86,6 +87,10 @@ export const WorkspaceList = () => {
   const { workspaces, refresh: refreshWorkspaces, loadingWorkspaces, loadingSubmissionStats } = useWorkspacesWithSubmissionStats()
   const [featuredList, setFeaturedList] = useState()
 
+  const { profile: { starredWorkspaces } } = useStore(authStore)
```

Comment:
It's unfortunate that the state management can't live entirely within the `WorkspaceStarControl` component.